bukuserver/api.py

```diff
@@ -16,7 +16,7 @@
 try:
     from response import Response
     from forms import ApiBookmarkCreateForm, ApiBookmarkEditForm, ApiBookmarkRangeEditForm, ApiTagForm
-except ImportError:
+except (ImportError, ModuleNotFoundError):
```

Comment:
[`ModuleNotFoundError` is a subclass of `ImportError`](https://docs.python.org/3.8/library/exceptions.html?highlight=modulenotfounderror#ModuleNotFoundError); this change doesn't actually have any effect. (You seem to have missed `During handling of the above exception, another exception occurred:` in the log, indicating that the actual issue happened _inside_ of the `except:` block.)